test/base.test.js

```diff
@@ -198,7 +196,7 @@ test('should set new session cookie if expired', async (t) => {
     fastify.addHook('onRequest', (request, reply, done) => {
       request.sessionStore.set(DEFAULT_SESSION_ID, {
         cookie: {
-          expires: Date.now() + 1000
+          expires: Date.now() - 1000
```

Comment:
This test was wrong! Interestingly, it was passing because sessionid wasn't provided so a new session was being made anyway.